openedx/core/djangoapps/waffle_utils/__init__.py

```diff
@@ -44,6 +44,20 @@
     with WAFFLE_SWITCHES.override(waffle.ESTIMATE_FIRST_ATTEMPTED, active=True):
         ...
 
+For long-lived flags, you may want to change the default for the flag from "off"
+to "on", so that it is "on" by default in devstack, sandboxes, or new Open edX
+releases, more closely matching what is in Production. This is for flags that
+can't yet be deleted, for example if there are straggling course overrides.
+
+    * WaffleFlag has a DEPRECATED argument flag_undefined_default that we don't
+    recommend you use any more. Although this can work, it is proven not ideal to
+    have a value that isn't immediately obvious via Django admin.
+
+    * At this time, the proper alternative has not been fully designed. Ideas
+    proposed include using migrations, managing "defaults" in a waffle.py inside
+    each app, or using management commands. If this need arises, the design
```

Comment:
Not sure what `managing "defaults" in a waffle.py inside each app` entails.  Would those defaults be exposed in the Django Admin?  May want to just remove this phrase instead of causing confusion with a currently half-baked idea.